src/bootstrap/src/core/build_steps/gcc.rs

```diff
@@ -0,0 +1,36 @@
+//! Retrieves information related to GCC backend.
+
+use crate::builder::Cargo;
+
+/// This retrieves the gccjit sha we use.
+pub(crate) fn detect_gccjit_sha() -> String {
+    // FIXME: This is absolutely ugly. Please find another way to do that.
+    let content =
+        include_str!("../../../../ci/docker/host-x86_64/dist-x86_64-linux/build-gccjit.sh");
```

Comment:
I'm not sure that putting it into a version file would be better since it would mean that we'd need to copy this file in all Dockerfiles using gcc backend. However it's a possibility, maybe less bad than the current one. No preference on my side.